### ISSUE TITLE
needs refactoring plan

### ISSUE THREAD

**Issue by vjcitn** [Bioconductor/biocViews#16]
this package is carrying a heavy weight.  we should have a special meeting to
discuss how it can be factored so that specific needs of BBS, html man pages etc.
can be handled in a more reasonable way?

**Comment by lshep** [comment]
sure. it will require careful testing while BBS is still active as this was attempted before by Marcel to remove things that broke the generation of the views and associated files for daily build reports.

**Comment by LiNk-NY** [comment]
If it breaks we fix it. Refactoring is vital to the ecosystem. 
FWIW, we may not need so much code in `biocViews` after transitioning to a GitHub-based BBS.

**Comment by lshep** [comment]
I agree but considering if VIEWS are not created we do not get updated landing pages on the current set up , I'm just saying we need to be careful because it can not be broken for an extended period of time.  And we shouldn't go in without trying to make sure that there are no interruptions
